### PR TITLE
Wc 25076 wc emails in hebrew

### LINKED ISSUE #25076: CSS fix for WooCommerce emails in `Hebrew` on mobile devices
**Describe the bug**
Emails are not displayed correctly in the Hebrew language:

https://cld.wthms.co/ok3FWD+

**To Reproduce**
1. Check if the default emails are working normally when the site language is set to English
    https://cld.wthms.co/8yaM5h+
    https://cld.wthms.co/I2UHaH+
2. Change language to Hebrew, and see that the emails don't fit in the screen on mobile devices
    https://cld.wthms.co/wR8JlL+
3. Update the translations and then again. No luck:
    https://cld.wthms.co/Kx0uLc+
    https://cld.wthms.co/ujBcK5+
    https://cld.wthms.co/2H7C8U+

**Expected behavior**
Without any changes to the template, the text should be wrapped, and content formatting should fit within the device screen

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
Any

### PR DESCRIPTION
**Pull request by mfs-mindsize**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR aims to resolve the bugs raised in WC issue:

"CSS fix for WooCommerce emails in `Hebrew` on mobile devices" #25076

[https://github.com/woocommerce/woocommerce/issues/25076](https://github.com/woocommerce/woocommerce/issues/25076)

Note: Best I could tell, the underlying markup + CSS "defects" impact more than rtl. Therefore, what you see in terms of changes might not be what you're expecting.  

Also, while I was under the hook I added escaping output where appropriate.

Closes #25076 

### How to test the changes in this Pull Request:

You'll have to be able to send test emails.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Fixed email header and CSS that was buggy for rtl emails (Issue #25076)

### CODE REVIEW

**Comment by rrennick** [comment]
@mfs-mindsize The `Closes #{issue-number}` is in the PR template because Github has an automation for that pattern which automatically closes the issue when the PR is merged. Can you edit your PRs and add the `Closes #` back?

**Comment by mfs-mindsize** [comment]
@rrennick - Updated. Sorry 'bout that. Now I know :)

File: templates/emails/email-header.php [hunk 1]
```diff
@@ -16,15 +16,15 @@
...
+	exit; // Exit if accessed directly.
 }
 
 ?>
 <!DOCTYPE html>
 <html <?php language_attributes(); ?>>
 	<head>
 		<meta http-equiv="Content-Type" content="text/html; charset=<?php bloginfo( 'charset' ); ?>" />
-		<title><?php echo get_bloginfo( 'name', 'display' ); ?></title>
+		<title><?php echo esc_attr( get_bloginfo( 'name', 'display' ) ); ?></title>
```
Comment (roykho) [inline templates/emails/email-header.php:27]: This is the wrong use of escaping here. Should be `esc_html()`. I see other areas in this PR doing the same as well.

**Review by vedanshujain — COMMENTED** [review]
I am not able to repro the issue, so have asked for additional details for helping reproduce in https://github.com/woocommerce/woocommerce/issues/25076#issuecomment-770662701.

Thanks for fixing the code standard violations along with the bug fix.

**Review by vedanshujain — CHANGES_REQUESTED** [review]
@mfs-mindsize I was able to reproduce this in android gmail, unfortunately does not looks like this PR fixes this issues. I can still reproduce the incorrect formatting after applying the patch.

Based on a cursory look, it seems like you have missed setting the max width on footer, which is causing the table to expand and then entire container div to expand. Can you add the max-width prop in footer as well like you did for body and header?

Thanks for helping out!

**Comment by samueljseay** [comment]
We are closing this pull request, as the last activity has been quite some time ago and we have not received a response from you. If you are still interested, please open another PR based on a recent version of the trunk branch.

**Pull request closed without merging**